src/emu.h

```diff
@@ -0,0 +1,111 @@
+#include <iostream>
+
+using byte_t = uint8_t;
+using word_t = uint16_t;
+template<typename T> bool IsPtrAligned(word_t ptr);
+template<> bool IsPtrAligned<byte_t>(word_t ptr) { return true; }
+//template<> bool IsPtrAlgined<word_t>(word_t ptr) { return !(ptr % sizeof(word_t)); }
```

Comment:
Explain the reason for commenting bottom function please.

Reply:
This part of code was not working when i last modified it.